yannickl88/features-bundle/1.0/etc/packages/features.yaml

```diff
@@ -0,0 +1,21 @@
+features:
+    tags:
+#        my.feature:
+#            request: ["beta", "on"]
+
+services:
```

Comment:
do we really need to have this file? see similar comment from @stof in sibling PR: https://github.com/symfony/recipes-contrib/pull/6#discussion_r113647743